jest.setup.js

```diff
@@ -0,0 +1,22 @@
+const { server } = require('./lib/mocks/server.js');
+const mockfs = require('mock-fs');
+
+// Establish API mocking before all tests.
+beforeAll(() => server.listen());
+
+beforeEach(() => {
+  mockfs({
+    '/home/user/meme.jpg': Buffer.from([8, 6, 7, 5, 3, 0, 9]),
+    '/home/user/lol.jpg': Buffer.from([9, 0, 3, 5, 7, 6, 8]),
+  });
```

Comment:
mock file system available in all tests. any fs.* method will think these are real. key == file, value == file contents